homework.md

```diff
@@ -44,37 +44,37 @@ Using CSS, how would I select every `<h1>` tag on the page?
 
 What is JavaScript?
 
-A programming language used on the internet
+A programming language for the website
```

Comment:
an object-oriented computer programming language commonly used to create interactive effects within web browsers.